composer.json

```diff
@@ -44,5 +50,7 @@
         "branch-alias": {
             "dev-master": "2.7-dev"
         }
-    }
+    },
+    "prefer-stable": true,
+    "minimum-stability": "dev"
```

Comment:
this must be removed. I want to run tests with ``"minimum-stability": "stable"`` by default, to be sure that we don't depend on something which does not have compatible stable versions (as it would hurt our users by forcing them to change their min stability)